src/TaskManager/TaskManager.cs

```diff
@@ -45,12 +48,14 @@ public class TaskManager : IHostedService, IDisposable, IMonaiService
         public TaskManager(
             ILogger<TaskManager> logger,
             IOptions<WorkflowManagerOptions> options,
-            IServiceScopeFactory serviceScopeFactory)
+            IServiceScopeFactory serviceScopeFactory,
+            IMinioAdmin minioAdmin)
         {
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
             _options = options ?? throw new ArgumentNullException(nameof(options));
 
             _serviceScopeFactory = serviceScopeFactory ?? throw new ArgumentNullException(nameof(serviceScopeFactory));
+            _minioAdmin = minioAdmin;
```

Comment:
Just sharing; instead of injecting the service in the constructor, it's also possible to get the service from the service scope factory.  This reduces a few changes in the unit test 🤣